tests/common/util.rs

```diff
@@ -1191,12 +1243,525 @@ impl UCommand {
     }
 }
 
+/// Stored the captured output in a temporary file. The file is deleted as soon as
+/// [`CapturedOutput`] is dropped.
+#[derive(Debug)]
+pub struct CapturedOutput {
+    output: tempfile::NamedTempFile,
+    current_file: File,
+}
+
+impl CapturedOutput {
+    /// Creates a new instance of CapturedOutput
+    fn new(output: tempfile::NamedTempFile) -> Self {
+        Self {
+            // TODO: do not reopen but use file pointer from output instead or current_file. That's
+            // one file descriptor less.
+            current_file: output.reopen().unwrap(),
+            output,
+        }
+    }
+
+    fn new_with(tempdir: &Rc<TempDir>) -> Self {
```

Comment:
This feels unnecessary, especially if we make this struct private and it's not used here.

Reply:
Yeah, this is a leftover. I wanted to place the temporary files from `CapturedOutput` into the temporary directory of the test scenario. It turned out, that some tests (I think some tests in `test_ls` were among them) rely on the content of the folder to be the way they set it up. It also works the way it is, so there's no need for this method anymore.